### ISSUE TITLE
Improve consistency of node naming and grouping in TAB menu

### ISSUE THREAD

**Issue by pojitonov** [stassius/StableHoudini#28]
**Is your feature request related to a problem? Please describe.**
Currently nodes have different approach for naming and grouping in TAB menu.
Grouping
- Some nodes stored under Stable Diffusion folder
- Some nodes are stored in Digital Assets folder

Naming in TAB Menu
- Some nodes don't have any prefix
- Some nodes has SD_ prefix
- One node has Sd _ prefix

![Snag_85d7ac](https://github.com/stassius/StableHoudini/assets/593064/2df6f5ed-a243-4eb6-b91c-f690eead6d97)
![Snag_85d20e](https://github.com/stassius/StableHoudini/assets/593064/e8b17369-825d-49cc-93fa-2acd1f477458)

Naming in Node Graph
- Some nodes don't have any prefix but uses Lower Case
- Some nodes don't have any prefix but uses Sentence Case
- Some nodes has SD_ prefix
- One node has Stable Diffusion _ prefix
![Snag_85cc61](https://github.com/stassius/StableHoudini/assets/593064/17ffa61a-bd1f-40fd-ac3b-c80a9a83f155)

**Describe the solution you'd like**
- To have one folder to store all related Nodes e.g. StableHoudini
- To have one convention for Node naming in TAB menu e.g. SH Node Name (Prefix + Sentence Case)
- To have one convention for node graph using Houdini convention to use lower case e.g. sh_node_name (prefix + lower case)

**Comment by stassius** [comment]
Agree, it bothers me as well.

**Comment by stassius** [comment]
Done. I changed most of the names, except for Stable Diffusion Dream internal name. As it's the main node and it will break existing projects if I change the definition.
From now on all SD nodes belong to the Stable Houdini folder, they have more or less consistent naming among toolset.
All image nodes - to the Image Processing folder, their naming is the same as in Labs nodes.
I changed definition of the Refresh Viewport Textures node, so if anyone happen to use it in their project, you'll need to Change Node Type or add it again.

I agree, that it would be great to conform the internal naming to the lowercase convention as well, but I don't want to break existing projects as Houdini doesn't have a way to just change the internal name of a node. I'll think about it.
